### PR TITLE
Resolve #12 Feature/html form

### LINKED ISSUE #12: Make basic styling for application
Get colour palette from modern CSS framework like Foundation or TW Bootstrap.

### PR DESCRIPTION
**Pull request by MariuszZalew**
Added SCSS file

### CODE REVIEW

File: src/css/style.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+$main-font: sans-serif;
+$bacground-green: #dff0d8;
+
```
Comment (benedyktdryl) [inline src/css/style.scss]: Do we need more than one line spacing?

File: src/css/style.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+body {
+	font-family: sans-serif;
+
+}
+
+
+.main {
+	background-color: #dff0d8;
+	width: 1200px;
```
Comment (benedyktdryl) [inline src/css/style.scss]: Can we make it responsive so it will stack one under another on smaller screens?

File: src/css/style.scss [hunk 3]
```diff
@@ -1,33 +1,31 @@
...
-		td {
-			padding: 10px;
-			text-align: center;
-		}
-	}
+    background-color: $pas-green;
+    margin: 20px auto;
+    padding: 20px;
+    max-width: 1200px;
+    .select-container {
```
Comment (Txwxcc) [inline src/css/style.scss]: If the DOM hierarchy will be changed, you would have to change SCSS also.

Comment (Txwxcc) [inline src/css/style.scss]: If the DOM hierarchy will be changed, you would have to change SCSS also.

File: src/css/style.scss [hunk 4]
```diff
@@ -1,33 +1,31 @@
...
+        flex-wrap: wrap;
+        justify-content: center;
+        width: 80%;
+        margin: 20px auto;
+        padding: 10px;
+        .item {
+            width: 300px;
+        }
+    }
+    .search-results {
```
Comment (Txwxcc) [inline src/css/style.scss]: Same here

**Comment by benedyktdryl** [comment]
**This start looks nice. Few things:**

- make things bigger a bit: inputs, fonts
- make table more visible, comparing to rest of application (different colors, grey border etc)
- on mobiles, inputs should be 90% size, centered horizontally
screen shot 2017-07-13 at 00 54 08

- on desktops, it looks like all inputs are aligned bit to left, can you fix this?
screen shot 2017-07-13 at 00 54 43

---

Let me know if you'll need help with that.

**Pull request merged**